pkg/systemd/services/services.jsx

```diff
@@ -50,12 +52,10 @@ import { useEvent, usePageLocation } from "hooks";
 
 const _ = cockpit.gettext;
 
-// As long as we have long-running superuser channels, we need to
-// reload the page when the access level changes.
-//
-superuser.reload_page_on_change();
-
-export const systemd_client = cockpit.dbus("org.freedesktop.systemd1", { superuser: "try" });
+export const systemd_client = {
+    system: cockpit.dbus("org.freedesktop.systemd1", { bus: "system", superuser: "try" }),
+    session: cockpit.dbus("org.freedesktop.systemd1", { bus: "session" }),
```

Comment:
Let's please call it `user`. These days it's really the user's bus/systemd instance, it gets shared across all the user's sessions. This also reflects `systemctl --user`.